whylog_vim/output_formater/output_aggregator.py

```diff
@@ -16,21 +21,26 @@ def get_content(self):
         # The same type as vim buffor.
         return self.output_lines
 
-    def create_button(self, callback_function, line=None):
+    def create_button(self, callback_function, function_meta, line=None):
         line = line or self._get_current_line_number()
-        self.buttons[line] = callback_function
+        self.function_lines[function_meta] = line
+        self.callbacks[line] = callback_function
 
     def set_default_callback_function(self, default_callback_function):
         self.default_callback_function = default_callback_function
 
+    def call_by_menu_function(self, menu_function):
+        self.call_button(self.function_lines[menu_function])
+
+    def get_line_number(self, function_meta):
```

Comment:
`function_lines` is public so you can access it directly when testing. Please remove this function for now. When (if...) keyboard shortcuts will be added in the future, we'll add it or do it differently.